docs/root/start/building.rst

```diff
@@ -26,12 +26,28 @@ Building Envoy has the following requirements:
   :ref:`this FAQ entry <faq_build_speed>` for more information on build performance.
 * These :repo:`Bazel native <bazel/repository_locations.bzl>` dependencies.
 
-Please see the linked :repo:`CI <ci/README.md>` and :repo:`Bazel <bazel/README.md>` documentation
-for more information on performing manual builds.
 Please note that for Clang/LLVM 8 and lower, Envoy may need to be built with `--define tcmalloc=gperftools`
 as the new tcmalloc code is not guaranteed to compile with lower versions of Clang.
 
 
+Windows Requirements
+--------------------
+
+Envoy has been ported and is now expected to work on Windows.
+
+Building Envoy has the following requirements:
+
+* A Windows (virtual) machine running on version 1903 (10.0.18362.1) and above.
+* The Windows 10 SDK, version 1803 (10.0.17134.12). Some features may require a newer SDK.
+* `Build Tools for Visual Studio 2019 <https://visualstudio.microsoft.com/downloads/#build-tools-for-visual-studio-2019>`_
+* These :repo:`Bazel native <bazel/repository_locations.bzl>` dependencies.
+
+Detailed instructions
+---------------------
+
+Please see :repo:`developer use of CI Docker images <ci/README.md>` and :repo:`building Envoy with Bazel <bazel/README.md>`
+documentation for more information on performing manual builds.
+
```

Comment:
I've made a couple of attempts to build on Windows and am still confused by what I need to do so. Is it possible to build Envoy for Windows on a Windows machine? 

If so, are their two methods?
1. natively in Windows via Bazel etc.
2. In Windows using Docker
 
Or do I have to build Envoy for Windows on a Linux machine with a Windows VM? Or are both possible? If I'm using Windows do I enter the commands in `cmd`, `powershell`, Git bash, some other Bash etc.?

Reply:
this does actually point to another confusing issue

ie - if you read these instructions and are wanting to build a linux container while working on a windows host (probs the most likely use case) then its not very clear that these are instructions for build a windows *image*

Reply:
(or maybe these are not instructions for building a windows image, and im really confused)

Reply:
I will try to address both the comments:

> I've made a couple of attempts to build on Windows and am still confused by what I need to do so.

I am happy to work with you offline to unblock you and we can take the learnings and put them on the docs.

> Is it possible to build Envoy for Windows on a Windows machine?

**Yes**, you can build and develop Windows on a Windows machine without the need for WSL or a linux VM or any linux technology.

> Natively in Windows via Bazel etc.

See https://github.com/envoyproxy/envoy/blob/main/bazel/README.md#building-envoy-with-bazel . If you encounter any issue when following the instructions or something is unclear please reach out or open an issue.

> In Windows using Docker

See https://github.com/envoyproxy/envoy/tree/main/ci#on-windows. As you point in the slack channel and this is something that **we should address** is that it is not obvious how you are supposed to run `./ci/run_envoy_docker.sh './ci/do_ci.sh bazel.release.server_only'`. The answer to that is from an MSYS2 bash terminal which you can get by running `C:\msys64\usr\bin\bash.exe` from a shell (powershell or cmd).

Also if you are using docker make sure that you are using Windows containers backend. See https://docs.docker.com/docker-for-windows/